src/htmlTransformers.js

```diff
@@ -44,6 +44,14 @@ export const transformAsides = content => {
   });
 };
 
+export const transformSpans = content => {
+  content('span[data-size]').each((_, span) => {
+    const s = span.attribs;
+    s.style = `font-size: ${s['data-size']};`;
```

Comment:
Vi har en egen klasse for dette `u-large-body-text` som vi burde bruke.

Så burde holde med noe slikt:
```
content =>
     content('span[data-size="large"]')
       .removeAttr('data-size')
       .addClass('u-large-body-text'),